src/components/Issues/IssueTable/index.tsx

```diff
@@ -1,35 +1,59 @@
 import React, { useState, useEffect } from 'react';
 import Pagination from '@material-ui/lab/Pagination';
-import { Box } from '@material-ui/core';
-
+import { Box, FormControl, InputLabel, Select, Input, Chip, MenuItem } from '@material-ui/core';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
 import qs from 'querystring';
 import IssueToolbar from './IssueToolbar';
 import IssueListItem from './IssueListItem';
 import service from '../../../service';
-import { IssueType } from '../issueTypes';
+import { IssueType, IProjectCardProps } from '../../../types';
 
-function IssueTable(): React.ReactElement {
+const useStyles = makeStyles(() =>
+  createStyles({
+    formControl: {
+      minWidth: 120,
+      maxWidth: 300,
+    },
+    chips: {
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    chip: {
+      margin: 2,
+    },
+  }),
+);
+interface ITableProps {
+  selectedProject: IProjectCardProps[];
+}
+function IssueTable(props: ITableProps): React.ReactElement {
   const [issues, setIssues] = useState<IssueType[]>([]);
   const [page, setPage] = useState<number>(1);
   const [totalPage, setTotalPage] = useState<number>();
-  const [selectedProject, setSelectedProject] = useState<string[]>([
-    '5fc8ed851f08a4a3b448d0e5',
-    '5fc8ed851f08a4a3b448d0e5',
-  ]);
+  const { selectedProject } = props;
   const handlePageChange = (event: React.ChangeEvent<unknown>, value: number) => {
     setPage(value);
   };
+
   useEffect(() => {
+    // if (!selectedProject) return;
     (async () => {
       const query = `?${qs.stringify({
         page,
-        projectId: selectedProject,
+        projectId: selectedProject.map((pj) => {
+          return pj._id;
+        }),
       })}`;
       const res = await service.getIssues(query);
-      setTotalPage(res.data.metaData[0].totalPage);
+      if (!res.data.data) {
```

Comment:
typescript를 쓰는 김에 ! 비교 보다는 명시적인 비교가 좋을것 같습니다.
```suggestion
      if (res.data.data === undefined) {
```